classes/class-excludes.php

```diff
@@ -195,10 +195,6 @@ function( $exclude ) {
 	 */
 	public function is_file_excluded( \SplFileInfo $file ) {
 
-		if ( ! $file->isFile() ) {
```

Comment:
@willmot I don't understand this 100%. I thought it only worked for files, not folders.. I've moved the original code logic from https://github.com/humanmade/backupwordpress/blob/master/admin/schedule-form-excludes.php#L233 into a separate function in https://github.com/humanmade/backupwordpress/pull/1078/files#diff-1852fb143db94269ade1ed75930ddc87R196

However, if I have that check the wholly excluded Folders get the "Exclude" action button, instead the correct "Excluded" text note. So I removed this check and all works as before.

I'm just a bit confused as to why.